tools/kompos/src/visualizations.ts

```diff
@@ -171,6 +78,9 @@ export function displayMessageHistory(msgHist: MessageHistoryMessage) {
   restart();
 }
 
+export function resetLinks (){
```

Comment:
Code style: `resetLinks() {` is preferred. (whitespace after parentheses instead of before)